src/runtime/auto-prompt-manager.js

```diff
@@ -272,7 +272,12 @@ export class AutoPromptManager {
       (clientConfig?.autoPromptConfig?.clientDisplayTrigger
         ?.displayDelaySeconds || 0) * SECOND_IN_MILLIS;
 
-    if (shouldShowAutoPrompt) {
+    const interventionDisplayedIsAutoPromptType =
+      this.interventionDisplayed_ === null ||
+      // Only need to check contributions because Subsciptions interventions
```

Comment:
I feel like `interventionDisplayed_` excluding subscription seems wrong. It can be unexpected as it's unclear in the name that this is only used for storing dismissals (for frequency-based triggering) and that's unneeded for subscription as there should be no suppression logic for subscription flow.

I think right thing to do is,
1) allow `interventionDisplayed_` to track any prompt type (remove the assumption that it's not subscription) 
2) allow dismissals for subscription prompt to be stored storeLastDismissal_.
3) Explicitly exclude subscription from the suppression logic that uses dismissed prompts.

Reply:
Updated the subscription flow to record `interventionDisplayed_`.